### PR TITLE
Added repeating and missing number problem

### PR DESCRIPTION
**Pull request by chitrasingla**
#317

### CODE REVIEW

**Comment by chitrasingla** [comment]
@tarun26091999 merge this pr
#317 issue is assigned to me

File: Code/C++/repeating_and_missing_number.cpp [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    {
+        // if element is at its position
+        if(a[i]==i+1 || a[i]==a[a[i]-1])
+            i++;
+        else
+            // if not swap
+            swap(a[i],a[a[i]-1]);
+    }
+
+    int r=0,m=0;
```
Comment (raksha009) [inline Code/C++/repeating_and_missing_number.cpp:41]: Add spaces around the operators =,< etc

File: Code/C++/repeating_and_missing_number.cpp [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+            break;
+        }
+    }
+
+    cout<<"Repeating number is "<<r<<endl;
+    cout<<"Missing number is "<<m;
+}
+/*
+	Testcases
+	Input :
```
Comment (raksha009) [inline Code/C++/repeating_and_missing_number.cpp:58]: Align testcases

File: Code/C++/repeating_and_missing_number.cpp [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+	Output : Repeating number is 1
+	Missing number is 5
+
+	Input :
+	6
+	1 4 5 2 3 4
+	Output : Repeating number is 4
+	Missing number is 6
+
+	Time complexity : O(n)
```
Comment (raksha009) [inline Code/C++/repeating_and_missing_number.cpp:70]: Specify n

**Review by raksha009 — CHANGES_REQUESTED** [review]

File: Code/C++/repeating_and_missing_number.cpp [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+5. Repeat the above steps till whole array is traversed.
+
+*/
+
+#include<bits/stdc++.h>
+using namespace std;
+
+int main()
+{
+    int n;
```
Comment (raksha009) [inline Code/C++/repeating_and_missing_number.cpp:22]: Specify what is n. Add appropriate comments everywhere.

File: Code/C++/repeating_and_missing_number.cpp [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+        // find the element which is not at its position
+        if(a[i] != i + 1)
+        {
+            r = a[i];
+            m = i + 1;
+            break;
+        }
+    }
+
+    cout<<"Repeating number is "<<r<<endl;
```
Comment (raksha009) [inline Code/C++/repeating_and_missing_number.cpp:53]: Add space around the operatir  <<

**Review by raksha009 — CHANGES_REQUESTED** [review]

**Review by rudrakshi99 — APPROVED** [review]

File: Code/C++/repeating_and_missing_number.cpp (hunk 5 above)
Comment (raksha009) [reply Code/C++/repeating_and_missing_number.cpp:54]: Plz Add space around the operator <<

**Review by raksha009 — CHANGES_REQUESTED** [review]

**Review by raksha009 — APPROVED** [review]

**Pull request merged**